src/Compilers/Core/Portable/MetadataReader/TypeNameDecoder.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using Microsoft.Cci;
```

Comment:
I think this namespace was not imported intentionally.

Reply:
It is, `CallingConvention` is a Cci type. If you prefer I can qualify that one instance and remove the import.

---
In reply to: [374347237](https://github.com/dotnet/roslyn/pull/41340#discussion_r374347237) [](ancestors = 374347237)

Reply:
> If you prefer I can qualify that one instance and remove the import.

Yes, I would prefer we do this as we are doing for other Cci type references in this file.

---
In reply to: [374393583](https://github.com/dotnet/roslyn/pull/41340#discussion_r374393583) [](ancestors = 374393583,374347237)